### ISSUE TITLE
Virus Detected

### ISSUE THREAD

**Issue by MacoDeVinx** [SeriousCache/UABE#582]
Hey, some users trying to download the UABE 3.0 Beta 1 will be met with a virus detected message and are unable to "ignore" or download further.

I downloaded this a few months ago without any issues but now when i try and download it, it will tell me a it contains viruses and are unable to continue it.

**Comment by presentHello** [comment]
Yeah virus total shows 30/64 issues...

**Comment by Wobbley** [comment]
How can you override it and open it anyways it literally just deleted the file on my Windows machine :|

**Comment by MacoDeVinx** [comment]
> How can you override it and open it anyways it literally just deleted the file on my Windows machine :|

Yeah, you can't, I downloaded mine months ago while it was working, but now it just says it's a virus and closes the download due to a virus, I guess.

I've run the 3.0 UABE through my anti-virus program and it doesn't come up with any virus so I am unsure why it says so upon downloading it now

**Comment by weslieallison** [comment]
I managed to temporarily disable my WIndows Defender as well as unblocking it in the zip properties.

**Comment by SuperO1k** [comment]
my stupid antivirus is restricting me from downloading it

**Comment by Ecorous** [comment]
I'm going to post this comment as hopefully a last message for people reading these threads. Please don't post anything after this unless it is useful to others reading these threads.

UABE does in fact appear to be a virus, so I would avoid using it for now. I would suggest using [UABEA](<https://github.com/nesrak1/UABEA/releases/latest>) which was and is a valid replacement before UABE was updated and/or opensourced. UABEA may (and is) still buggy, so be aware of that. Again I would entirely stay away from UABE as there has been no proof that it is not a virus, where **many** pieces of antivirus software (including Windows Defender) pick it up as malicious. Until this has been resolved, stay well clear of UABE, and if you need a replacement, use [UABEA](<https://github.com/nesrak1/UABEA/releases/latest>)

**Comment by MacoDeVinx** [comment]
Actually, I would recommend 2.2 b or other 2.2 version of [UABE](https://github.com/SeriousCache/UABE/releases/tag/2.2stabled).
It is only the new UABE version [3.0 beta 1] that has virus detection on it.

Personally not a fan of the UABEA layout and it's also missing a few key features for modding, such as mesh modding and audio files, the only thing you can do is textures.

By using 2.2 you can do whatever but only for games with Unity 2019.2 though.

[Follow up on Ecorous message.]

**Comment by Crayder** [comment]
Well 2.2, UABEA, and this all three give virus warning
